### PR TITLE
PODS-3034: make H1 dynamic for all current and previous address pages

### PR DESCRIPTION
**Pull request by Ashwani1403**
PODS-3034: make H1 dynamic for all current and previous address pages

### CODE REVIEW

File: app/controllers/register/trustees/individual/TrusteeAddressController.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ class TrusteeAddressController @Inject()(
               postCall(mode, Index(index), srn),
```
Comment (madhuranakate) [inline app/controllers/register/trustees/individual/TrusteeAddressController.scala]: Please use toggle for TrusteeNameId vs TrusteeDetailsId

File: app/controllers/register/trustees/individual/TrusteePreviousAddressController.scala [hunk 2]
```diff
@@ -67,7 +67,7 @@ class TrusteePreviousAddressController @Inject()(
               postCall(mode, Index(index), srn),
```
Comment (madhuranakate) [inline app/controllers/register/trustees/individual/TrusteePreviousAddressController.scala]: Please use toggle for TrusteeNameId vs TrusteeDetailsId

File: conf/messages.en [hunk 3]
```diff
@@ -419,6 +421,7 @@ messages__error__paye_invalid = Enter an employer PAYE number using only numbers
 messages__error__has_paye_establisher = Select whether the partnership has an employer PAYE number
 
 messages__establisher_individual_address__title = Establisher’s address
+messages__establisher_individual_confirmAddress__heading = Confirm {0}’s address
```
Comment (madhuranakate) [inline conf/messages.en]: This message is duplicate of the one on line 54, can we reuse that?

Comment (Ashwani1403) [reply conf/messages.en]: added condition for H&S. please check

File: app/controllers/register/trustees/individual/TrusteeAddressController.scala (hunk 1 above)
Comment (Ashwani1403) [reply app/controllers/register/trustees/individual/TrusteeAddressController.scala]: added condition for H&S, please check now

**Pull request merged**